### DISCUSSION TITLE
Importing twitter followers

### DISCUSSION THREAD

**Discussion by worbgge** [jonjomckay/fritter#244] [Q&A] [answered]
Hello, I haven't been able to find the answer anywhere so i thought i would start an issue but how can i import data? i have a .txt file of loads of accounts that i want to follow is there anything else i need to do since when i press import and the txt file it doesn't do anything so is there a specific thing? thank you

**Comment by jonjomckay** [comment, marked as answer]
Right now, Fritter supports importing subscriptions from an existing (public) Twitter account, using the Import/Export button on the Subscriptions tab. You can also import from a previous export of data from Fritter.

If you have a text file containing subscriptions, you might be able to import it by converting it to this format, and putting it into a JSON file:

```json
{
    "subscriptions": [
        {
            "id": "the-twitter-user-id-12345",
            "screen_name": "username",
            "name": "Display Name",
            "verified": 1
        }
    ]
}
```

The minimum amount of correct data only needs to be the `id` field - the other fields can be anything, and once imported, a refresh will fetch the correct profile data.
